File: Ano 1/Worspace JavaBasico/IF/src/Desafio3.java
import java.util.Scanner;

public class Desafio3 {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		System.out.println("Informe um número de 1-7 (SEMANA) Ex.:(5)");
		int numero = sc.nextInt();
		
		if(numero == 1) {
			System.out.println("Domingo");
		} else if(numero == 2) {
			System.out.println("Segunda-feira");
		} else if(numero == 3) {
			System.out.println("Terça-feira");
		} else if(numero == 4) {
			System.out.println("Quarta-feira");
		} else if(numero == 5) {
			System.out.println("Quinta-feira");
		} else if(numero == 6) {
			System.out.println("Sexta-feira");
		} else if(numero == 7) {
			System.out.println("Sábado");
		}

	}

}

File: Ano2/Ead/OT 02- TYPESCRIPT/desafio3.ts
enum DiasDaSemana {
    Domingo,
    Segunda,
    Terca,
    Quarta,
    Quinta,
    Sexta,
    Sabado
}

function mensagemDia(dia: DiasDaSemana): void {
    switch (dia) {
        case DiasDaSemana.Segunda:
            console.log("Hoje é segunda-feira, início da semana!");
            break;
        case DiasDaSemana.Sexta:
            console.log("Hoje é sexta-feira, quase fim de semana!");
            break;
        default:
            console.log(`Hoje é ${DiasDaSemana[dia]}.`);
    }
}

mensagemDia(DiasDaSemana.Sexta); // Hoje é sexta-feira, quase fim de semana!
